test/python/compiler/test_transpiler.py

```diff
@@ -1646,7 +1648,10 @@ def test_no_infinite_loop(self, optimization_level):
     @data(0, 1, 2, 3)
     def test_transpile_preserves_circuit_metadata(self, optimization_level):
         """Verify that transpile preserves circuit metadata in the output."""
-        circuit = QuantumCircuit(2, metadata={"experiment_id": "1234", "execution_number": 4})
+        metadata = {"experiment_id": "1234", "execution_number": 4}
+        name = "my circuit"
+        circuit = QuantumCircuit(2, metadata=metadata.copy(), name=name)
+        circuit.name = "my circuit"
```

Comment:
Isn't `circuit.name = "my circuit"` redundant? Passing `name=name` to the `__init__` function sets the attribute `name` already.

Reply:
Yeah, thanks - I must have left that line in when I modified how I originally wrote the test.  Done in e2d9967ae2.